tutoroars/plugin.py

```diff
@@ -142,6 +142,239 @@
             },
         ),
         ("SUPERSET_TALISMAN_ENABLED", True),
+        ("SUPERSET_EXTRA_ASSETS", {
```

Comment:
@bmtcril I think we can use a patch for this, defining this large YAML in the **config.yaml** file would be a pain